llarp/router/router.cpp

```diff
@@ -150,15 +151,12 @@ namespace llarp
     builders.push_back(services["default"]["buildStats"]);
 
     // Iterate over all items on this array to build the global pathStats
-    uint64_t paths = 0;
     uint64_t success = 0;
     uint64_t attempts = 0;
     for (const auto& builder : builders)
     {
       if (builder.is_null())
         continue;
-      if (builder["length"].is_number())
```

Comment:
replacing "length" with "numPaths" in this hunk would fix this issue much better.